roles/rsyslog/templates/input_basics.j2

```diff
@@ -3,17 +3,21 @@ module(load="imklog" permitnonkernelfacility="on")
 {% endif %}
 {% if __rsyslog_input.use_imuxsock | d(false) | bool %}
 module(load="imuxsock"    # provides support for local system logging (e.g. via logger command)
+       SysSock.RateLimit.Interval="{{ __rsyslog_input.ratelimit_interval | d(0) }}"
+{% if __rsyslog_input.ratelimit_interval is defined and __rsyslog_input.ratelimit_interval > 0 %}
```

Comment:
```suggestion
{% if __rsyslog_input.ratelimit_interval | d(0) > 0 %}
```
maybe this will make the ansible-core 2.16 runs happy?

and also - before we were setting `SysSock.RateLimit.Interval` to the `__rsyslog_input.ratelimit_burst` value - that's not good . . . thanks for finding this